Sources/HipMobileUI.Android/MainActivity.cs

```diff
@@ -87,11 +87,11 @@ protected override void OnCreate(Bundle bundle)
             CrashManager.Register(this, keyProvider.GetKeyByName("hockeyapp.android"));
 
             // init forms and third party libraries
-            CachedImageRenderer.Init();
+            CachedImageRenderer.Init(enableFastRenderer: true);
```

Comment:
Could this be the culprit for blurry images?

Reply:
Probably, we need to test this